packages/account/src/account.test.ts

```diff
@@ -307,13 +308,11 @@ describe('Account', () => {
     const response = await sender.transfer(receiver.address, 1, baseAssetId, {
       gasLimit: 10_000,
     });
-    await response.wait();
```

Comment:
Why have all the balance checks in these tests been removed? They don't seem relevant to this PR

Reply:
cc @Torres-ssf

Reply:
@maschad The related tests were failing because the balance was being compared to pre-defined values. Since some gas instructions values changed, the fee changed, resulting in the test failing.

I've simplified all these tests as the balance validation part does not add any relevance to the test case.

Reply:
Thanks for clarifying